UITests/Components/IssuerList/__Snapshots__/IssuerListComponentUITests/testStartStopLoading.initial_state.txt

```diff
@@ -0,0 +1,49 @@
+<UIView; frame = (0 0; 390 844); autoresize = W+H; backgroundColor = <UIDynamicSystemColor; name = systemBackgroundColor>; layer = <CALayer>>
```

Comment:
This file is used instead of screenshots? 🤔

Reply:
yes, using this technique might solve the issue with the spinner animation.